### PR TITLE
 Issue #1904: Fields of text output are not explained

### PR DESCRIPTION
**Pull request by fschueller**
Fixes https://github.com/SUSE/machinery/issues/1904, in which 'N/A' fields are not further explained.

### CODE REVIEW

**Comment by thardeck** [comment]
I would only show the message when there are N/A elements, otherwise it might be annoying. Or making it optional somehow.

**Comment by greygoo** [comment]
LGTM

File: lib/renderer.rb [hunk 1]
```diff
@@ -266,4 +266,9 @@ def indent
     yield
     @indent -= 2
   end
+
+  def na_note(s)
```
Comment (thardeck) [inline lib/renderer.rb]: I would use a speaking variable name.

File: spec/support/system_description_factory.rb [hunk 2]
```diff
@@ -631,6 +649,25 @@ def build_description(name, store, options)
...
   EOF
 
+  EXAMPLE_SCOPES["users_without_comment"] = <<-EOF.chomp
+    "users": {
+      "_attributes": {},
+      "_elements": [
+        {
+          "name": "bin",
+          "password": "x",
+          "uid": null,
```
Comment (thardeck) [inline spec/support/system_description_factory.rb]: Is this case possible? I do not think that a user without a uid is possible.

File: lib/renderer.rb (hunk 1 above)
Comment (fschueller) [inline lib/renderer.rb]: I followed the example of other methods, but I can change it nevertheless.

File: spec/support/system_description_factory.rb (hunk 2 above)
Comment (fschueller) [inline spec/support/system_description_factory.rb]: The code at least covers this case, in ``` plugins/users/users_renderer.rb```, line 25.

**Pull request closed without merging**